metadata.rb

```diff
@@ -1,8 +1,7 @@
-name 'base-win2012-hardening'

```

Comment:
I think there is no need to update that file. We updated the name of the cookbook. We should bump the version to 1.x though since this is adding a lot of new features